Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/Controller.mo

```diff
@@ -131,15 +147,17 @@ boolean relay switch output <code>yOn</code>, and the control error
 </p>
 <ul>
 <li>
-<code>yErr = u_s - u_m</code>,
+<code>yErr = u_m - u_s</code>,
```

Comment:
It would be better to change the documentation as:
- if the parameter <code>reverseActing = false</code>, then,
  - <code>yErr = u_m - u_s</code>,
  - if yErr < -deaBan and trigger is true, ...
  - if ...
  - ...
- else if  the parameter <code>reverseActing = true</code>, then,
  -   - <code>yErr = u_s - u_m</code>,
  - if yErr < -deaBan and trigger is true, ...
  - if ...
  - ...

Reply:
As suggested, the model doc is updated.